mapBuilder/controllers/default.classic.php

```diff
@@ -1,39 +1,40 @@
 <?php
+
 /**
-* @package   lizmap
-* @subpackage mapBuilder
-* @author    3liz
-* @copyright 2018-2020 3liz
-* @link      http://3liz.com
-* @license   Mozilla Public License : http://www.mozilla.org/MPL/
-*/
-
-class defaultCtrl extends jController {
-    /**
-    *
-    */
-    function index() {
+ * @author    3liz
+ * @copyright 2018-2020 3liz
+ *
+ * @see      http://3liz.com
+ *
+ * @license   Mozilla Public License : http://www.mozilla.org/MPL/
+ */
+class defaultCtrl extends jController
+{
+    public function index()
+    {
         // Access control
-        if( !jAcl2::check('mapBuilder.access')){
-          $rep = $this->getResponse('redirect');
-          $rep->action = 'view~default:index';
-          jMessage::add(jLocale::get('view~default.repository.access.denied'), 'error');
-          return $rep;
+        if (!jAcl2::check('mapBuilder.access')) {
+            $rep = $this->getResponse('redirect');
+            $rep->action = 'view~default:index';
+            jMessage::add(jLocale::get('view~default.repository.access.denied'), 'error');
+
+            return $rep;
         }
 
-        $rep = $this->getResponse('html', true);// true désactive le template général
+        $rep = $this->getResponse('html', true); // true désactive le template général
```

Comment:
```suggestion
        $rep = $this->getResponse('html', true); // true disables the global/main template
```
?
To switch to english

Reply:
ofc !